src/documents/IoCReferences.md

```diff
@@ -21,12 +24,14 @@ References always take the syntactic form `{context-name}.some.path.segments` -
             <td>
                 <ul>
                     <li>
-                        <code>&lt;componentRef&gt;</code>is a reference to a component via one of its context names. It may be:
+                        <code>&lt;componentRef&gt;</code> is a reference to a component via one of its context names. It may be:
                         <ul>
-                            <li><code>that</code> to reference the current component e.g. <code>{that}</code></li>
+                            <li><code>that</code>, to reference the current component e.g. <code>{that}</code></li>
                             <li>the fully qualified name of the component's type or one of its <a href="ComponentGrades.md">grade names</a> e.g. <code>{fluid.pagedTable}</code></li>
                             <li>the <strong>short name</strong> of the component's type or one of its <strong>gradeNames</strong>, i.e. the last segment of the fully namespaced name, e.g. <code>{pagedTable}</code></li>
                             <li>the component's <strong>member name</strong>, i.e. the name used when defining a subcomponent in a components block</li>
+                            <li>an entire <code>&lt;reference&gt;</code> string in itself. This is useful in highly dynamic cases where the context name should be determined from some other options material, e.g. <code>{{that}.options.targetContext}</code>. This
+                            feature is new in Infusion 2.0.
```

Comment:
@amb26 we talked about this today at the Floe Planning meeting. We've decided to hold off on flagging items as new/changed in the Infusion 2.0 docs. We'll start adding these to the documentation after 2.0. With that in mind, it was also decided that the "new in Infusion 2.0" tags you've added in this PR should also be removed, to avoid implying that other parts of the docs aren't new.